cmd/syft/internal/test/integration/package_deduplication_test.go

```diff
@@ -1,13 +1,10 @@
-//go:build !arm64
```

Comment:
I was able to make this cross-platform, which makes the fixture build/sync logic a little simpler (it's the same for all platforms)